.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please search or post to our Discourse site: https://discourse.julialang.org/c/domain/opt. We use
+the GitHub issue tracker for bug reports and feature requests only.
+
+If the bug you're experiencing is solver-specific (e.g., it only happens when you use Gurobi),
+the issue is best raised at the relevent solver-wrapper repository. If you are unsure, you should
+raise the problem on Discourse.
+
+If you've confirmed that the behaviour is a bug, and it relates to JuMP, this is the right place.
```

Comment:
It's our job to confirm if something it's a bug. Maybe say "if you are reasonably confident that ...". Also repeat that JuMP feature requests are valid issues.